binding.cc

```diff
@@ -390,8 +390,7 @@ namespace zmq {
 
   void
   Socket::UV_PollCallback(uv_poll_t* handle, int status, int events) {
-    assert(status == 0);
-
+    if (status != 0) NanThrowError("I/O status: socket not ready !=0 ");
```

Comment:
ya ok, this is the thread i meant to write back on in the other replies to the xpub/xsub issue i made today. i'll see if i can edit or delete those, really dumb of me..